tests/model_example/generate_expected.py

```diff
@@ -33,8 +40,16 @@
     image_bus_annotated = results_bus[0].plot(conf=True)
     image_zidane_annotated = results_zidane[0].plot(conf=True)
 
+	# Generate expected
+    bounding_box_bus = results_bus[0].boxes.xyxy.detach().cpu().numpy()
+    bounding_box_zidane = results_zidane[0].boxes.xyxy.detach().cpu().numpy()
+
     # Save image
-    cv2.imwrite("tests/model_example/bus_annotated.png", image_bus_annotated)
-    cv2.imwrite("tests/model_example/zidane_annotated.png", image_zidane_annotated)
+    cv2.imwrite(f"{SAVE_PATH}/bus_annotated.png", image_bus_annotated)
+    cv2.imwrite(f"{SAVE_PATH}/zidane_annotated.png", image_zidane_annotated)
+    
+	# Save expected to text file
+    np.savetxt(f"{SAVE_PATH}/bounding_box_bus.txt", bounding_box_bus)
+    np.savetxt(f"{SAVE_PATH}/bounding_box_zidane.txt", bounding_box_zidane)
```

Comment:
Make both save paths global constants. You can still have a save directory global constant which is used to construct each of these paths.